src/rpc/client.cpp

```diff
@@ -225,11 +225,14 @@ class CRPCConvertTable
 public:
     CRPCConvertTable();
 
-    bool convert(const std::string& method, int idx) {
-        return (members.count(std::make_pair(method, idx)) > 0);
+    /* Return value as UniValue, and first parse it if it is a non-string parameter */
+    UniValue ParseIfNonStringParam(const std::string& value, const std::string& method, int idx) {
```

Comment:
clang format new code?

Reply:
Also, could rename the function to just `UniValue Convert(...`, given that you removed `bool convert(...` and the caller doesn't care about the implementation details here, only that this turns a value into `UniValue`.

Reply:
> clang format new code?

done

> Also, could rename the function to just UniValue Convert(...

Agreed with the caller not caring about implementation details, but I've now adopted @ryanofsky's [suggested](https://github.com/bitcoin/bitcoin/pull/26506#discussion_r1044667023) `ArgToUniValue` since it's more clear than `Convert()` wrt what kind of conversion we're doing (and also slightly improved function parameter naming)